runtime/oti/j9nonbuilder.h

```diff
@@ -3573,8 +3573,19 @@ typedef struct J9HookedNative {
 	UDATA userdata;
 } J9HookedNative;
 
-/* @ddr_namespace: map_to_type=J9ClassLoader */
+typedef struct J9RAMClassFreeLists {
+	struct J9RAMClassFreeListBlock *ramClassTinyBlockFreeList;
+	struct J9RAMClassFreeListBlock *ramClassSmallBlockFreeList;
+	struct J9RAMClassFreeListBlock *ramClassLargeBlockFreeList;
+} J9RAMClassFreeLists;
+
+typedef struct J9RAMClassUDATABlockFreeList {
+	UDATA *ramClassSub4gUDATABlockFreeList;
```

Comment:
I think @theresa-m previous comment was to make the UDATAblockFreeList part of `J9RAMClassFreeLists`